packages/create-svelte/templates/default/package.json

```diff
@@ -4,7 +4,8 @@
 	"scripts": {
 		"dev": "svelte-kit dev",
 		"build": "svelte-kit build --verbose",
-		"start": "svelte-kit start"
+		"start": "svelte-kit start",
+		"deploy": "ADAPTER=@sveltejs/adapter-vercel npm run build && vercel deploy --prebuilt"
```

Comment:
Was this checked in by accident? I don't think it'd work unless you also add this to the default `package.json`

Reply:
yes, this was just for testing